examples/ethernet/ethernet/main/Kconfig.projbuild

```diff
@@ -0,0 +1,17 @@
+menu "Example Configuration"
+
+config PHY_TLK110
```

Comment:
Ok.  I looked at all of the Kconfig file examples under the examples tree and didn't see an example of that.  I'll make the change.